src/GovUk.Education.ExploreEducationStatistics.Data.Seed/Services/SeedService.cs

```diff
@@ -31,44 +41,140 @@ public void Seed()
             var stopWatch = new Stopwatch();
             stopWatch.Start();
 
-            try
+            foreach (var theme in SamplePublications.Themes)
             {
-                _context.ChangeTracker.AutoDetectChangesEnabled = false;
-
-                foreach (var theme in SamplePublications.Themes)
+                foreach (var topic in theme.Topics)
                 {
-                    _logger.LogInformation("Updating Theme {Theme}", theme.Title);
-                    var updated = _context.Theme.Update(theme).Entity;
-                    _context.SaveChanges();
+                    foreach (var publication in topic.Publications)
+                    {
+                        foreach (var release in publication.Releases)
+                        {
+                            foreach (var subject in release.Subjects)
+                            {
+                                _logger.LogInformation("Seeding Subject {Subject}", subject.Name);
+
+                                Release r = new Release
```

Comment:
I'm not sure why we are building the Release/Publication/Topic/Theme here because they are already build in SamplePublications.

Something like this?

```
                    var subjects = SamplePublications.Themes
                        .SelectMany(theme => theme.Topics)
                        .SelectMany(topic => topic.Publications)
                        .SelectMany(publication => publication.Releases)
                        .SelectMany(release => release.Subjects);

foreach (var subject in subjects) {
  var file = SamplePublications.SubjectFiles.GetValueOrDefault(subject.Id);
  StoreFiles(subject.Release, file, subject.Name);
  Seed(file + ".csv", subject.Release);
}
```

Reply:
Think I tried that initially but it does create the collections in the correct way. Will try again though.